index.html

```diff
@@ -256,6 +261,26 @@ <h3>The <dfn><code>VisualViewport</code></dfn> interface</h3>
                 Although it is referred to as the pinch-zoom scaling factor, it can be affected through means other than pinch-zooming. e.g. When the UA centers and zooms in a focused edit box.
               </div>
             </dd>
+            <dt><dfn>segments</dfn></dt>
+            <dd>
```

Comment:
Per discussion, can we add a `Note:` here about this API being experimental or that it is an addition to the 'base APIs' that are currently shipping?